runner/app/pipelines/live_video_to_video.py

```diff
@@ -107,7 +107,10 @@ class PipelineStatus(BaseModel):
             raise ConnectionError(f"Failed to get status: {e}")
 
     def start_process(self, **kwargs):
-        cmd = [str("/workspace/miniconda3/envs/comfystream/bin/python"), str(self.infer_script_path)]
+        if kwargs.get("pipeline") == "comfyui":
+            cmd = ["conda", "run", "--no-capture-output", "-n", "comfystream", "python", str(self.infer_script_path)]
+        else:
+            cmd = [str(self.infer_script_path)]
```

Comment:
Does this work without `python` as the first element? Before the #479 it was

https://github.com/livepeer/ai-runner/blob/aa1700685b3a4ef3e77bbbf046fa7cf97c407285/runner/app/pipelines/live_video_to_video.py#L110

Reply:
Probably not, let me check that again

Reply:
Rebased, fixed and tested noop pipeline.

Reply:
I changed to use "python". Otherwise it may not work if you don't have correct file permissions. In my case it was missing "x", so it didn't work.